[0m19:52:35.497214 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10e8a5f00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1109e7670>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1109e78e0>]}


============================== 19:52:35.507927 | 094db8ca-e866-4d58-8b6c-53f25ff36adf ==============================
[0m19:52:35.507927 [info ] [MainThread]: Running with dbt=1.6.9
[0m19:52:35.509132 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/jessicaschulz/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init get_better', 'send_anonymous_usage_stats': 'True'}
[0m19:52:35.510198 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/Users/jessicaschulz/.dbt')}
[0m19:52:35.510836 [info ] [MainThread]: Creating dbt configuration folder at 
[0m19:52:35.512655 [debug] [MainThread]: Starter project path: /Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/dbt/include/starter_project
[0m19:52:35.522909 [info ] [MainThread]: 
Your new dbt project "get_better" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m19:52:35.523631 [info ] [MainThread]: Setting up your profile.
[0m19:57:05.063894 [error] [MainThread]: Encountered an error:

[0m19:57:05.086058 [error] [MainThread]: Traceback (most recent call last):
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/dbt/cli/requires.py", line 87, in wrapper
    result, success = func(*args, **kwargs)
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/dbt/cli/requires.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/dbt/cli/main.py", line 484, in init
    results = task.run()
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/dbt/task/init.py", line 308, in run
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/dbt/task/init.py", line 179, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/dbt/task/init.py", line 163, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/dbt/task/init.py", line 113, in generate_target_from_input
    numeric_choice = click.prompt(prompt_msg, type=click.INT)
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
  File "/Users/jessicaschulz/mambaforge/envs/dbt16/lib/python3.10/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m19:57:05.090114 [debug] [MainThread]: Command `dbt init` failed at 19:57:05.089357 after 269.64 seconds
[0m19:57:05.092563 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10e8a5f00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1110be320>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1110bf0a0>]}
[0m19:57:05.096462 [debug] [MainThread]: Flushing usage events
